# Indicate the usage of Ember Components (`ember-observer/no-ember-components`)

<!-- end auto-generated rule header -->
Used by Ember Observer to determine if addons are using Ember components.

## Examples

Examples of **incorrect** code for this rule:

```javascript
  import Component from '@ember/component';
```
