# Indicates use of glimmer components (`ember-observer/no-glimmer-components`)

<!-- end auto-generated rule header -->

Indicate use of Glimmer Components
Used by Ember Observer to determine if addons are using glimmer components.

## Examples

Examples of **incorrect** code for this rule:

```javascript
  import Component from '@glimmer/component';
```
